guides/integrate-nextauth.md

```diff
@@ -0,0 +1,327 @@
+---
+title: NextAuth
+sidebar_position: 6
+---
+
+import SetupJavaScript from '/docs/workflows/_sdk-setup/_setup-javascript.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+import AppSchemeCaution from '/docs/workflows/\_app-scheme-caution.mdx';
+import InvocationTip from '/docs/workflows/\_invocation-type-tip.mdx';
+import InvocationDiagram from '/docs/platform-overview/\_invocation-url-diagram.mdx';
+
+# Integrate Beyond Identity Passwordless Authentication into NextAuth
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a Next application that uses NextAuth.
+
+This guide will cover:
+
+- How to install the JavaScript SDK
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+- How to configure the OIDC NextAuth plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## NextAuth
+
+- [NextAuth Introduction](https://authjs.dev/getting-started/introduction)
+- [NextAuth Guides](https://authjs.dev/guides)
+- [Beyond Identity Provider](https://authjs.dev/reference/core/providers_beyondidentity)
+
+### Example
+
+```javascript
+import { Auth } from "@auth/core"
+import BeyondIdentity from "@auth/core/providers/beyondidentity"
+
+const request = new Request("https://example.com")
+const response = await Auth(request, {
```

Comment:
I wonder if we would actually use the `request` in this guide? I think I would need to try adding the provider to Acme to see. It seems like we wouldn't need to directly set this though.

Reply:
I think it is only needed as the first parameter of the Auth constructor, but this could be inline